gatk-launch

```diff
@@ -109,6 +110,12 @@ def main(args):
             del args[i] #remove spark target
             del args[i] #and its parameter
 
+        sparkSubmit = getValueForArgument(args, "--sparkSubmit")
```

Comment:
We're doing this ugly hack quite a few times now.  Can we abstract a function that does it? 

(as the originator of the hideous hack I do feel ashamed... at some point we should probably switch to using a proper argument parsing system... or integrate gatk-launch into hellbender so we can use our own argument parsing system...)